integration-tests/src/test/resources/anchor-config-integration.yaml

```diff
@@ -0,0 +1,249 @@
+#######################################################################
+## Anchor Platform - Integration Test Config
+#######################################################################
+
+##################################
+## Required System Configuration
+##################################
+version: 0.2
+
+data:
+  url: jdbc:h2:mem:test
+  options:
+    spring.jpa.properties.hibernate.dialect: org.hibernate.dialect.H2Dialect
+    spring.liquibase.enabled: false
+
+# The anchor server callback API endpoint for customer, fee, quote, and unique url integration.
+callbackApi:
+  baseUrl: https://localhost:8080
+  authType: JWT_TOKEN
+
+stellarNetwork:
+  # Use `TESTNET` or `PUBNET` network
+  network: TESTNET
+  horizonUrl: https://horizon-testnet.stellar.org
+  networkPassphrase: 'Test SDF Network ; September 2015'
+
+#################################
+## Optional System Configuration
+##################################
+events:
+  enabled: true
+  publisherType: kafka
+  options:
+    bootstrapServer: localhost:29092
+
+paymentObserver:
+  enabled: true
+
+logging:
+  level: INFO
+
+metrics:
+  enabled: true
+
+language: en
+
+#######################
+## SEP Configuration
+#######################
+sep1:
+  enabled: true
```

Comment:
```suggestion
  enabled: false
```

Reply:
This is the config file used for integration tests. Do we want to have these disabled in that context?